SignalServiceKit/src/Messages/Interactions/TSOutgoingMessage.m

```diff
@@ -761,7 +761,7 @@ - (OWSSignalServiceProtosDataMessageBuilder *)dataMessageBuilder
             case TSGroupMessageNew: {
                 if (gThread.groupModel.groupImage != nil && self.attachmentIds.count == 1) {
                     attachmentWasGroupAvatar = YES;
-                    [groupBuilder setAvatar:[self buildProtoForAttachmentId:self.attachmentIds[0] filename:nil]];
```

Comment:
this is the fix - 'buildProtoForAttachmentId' got moved to TSAttachmentStream method, but I failed to remove the header definition so this still compiled even though the method didn't exist.